README.md

```diff
@@ -91,9 +91,20 @@ $ inv run.gunicorn
 
 The site will be accessible on localhost port 9876 (it's a countdown!).
 
+The full sequence of commands to start running a local version of the site:
+```
+inv run.mysql
```

Comment:
do you want to add  `workon pyspaceship` and `pip install -r requirements.txt`

Reply:
yes good idea for completeness